src/summary/documentation/vitals-card.component.tsx

```diff
@@ -7,6 +7,7 @@ import { formatDate } from "./dimension-helpers";
 import { createErrorHandler } from "@openmrs/esm-error-handling";
 import { useCurrentPatient } from "@openmrs/esm-api";
 import SummaryCardFooter from "../cards/summary-card-footer.component";
+import { link } from "fs";
```

Comment:
Why do we require this import `import { link } from "fs"`

Reply:
@jonathandick ?

Reply:
Not sure where that came from. Was it me that added? Either way, removing.